packages/react-server-dom-webpack/src/ReactFlightClientConfigWebpackBundler.js

```diff
@@ -111,7 +111,33 @@ export function resolveServerReference<T>(
 // in Webpack but unfortunately it's not exposed so we have to
 // replicate it in user space. null means that it has already loaded.
 const chunkCache: Map<string, null | Promise<any>> = new Map();
-const asyncModuleCache: Map<string, Thenable<any>> = new Map();
+
+function requireAsyncModule(id: string): null | Thenable<any> {
```

Comment:
don't love the name. My intuition is this has some parity with the `requireModule` function below but it's really about preloading only. maybe `loadAsyncModule`?

Reply:
But then it sounds it should be using the `load` API. It doesn't load the module itself, it requires it which might load something that isn't even a module or nothing at all.

The confusing part here is that there's two naming conventions `requireAsyncModule` is kind of a webpack/cjs thing, where as `requireModule` is more of a Flight concept so it names it. We should probably rename `requireModule` to something else because "require" is kind of CJS specific name for that concept.